CHANGELOG.md

```diff
@@ -1,16 +1,52 @@
-# Changelog
+# CHANGELOG
 
-## [Unreleased](https://github.com/CosmWasm/cw-storage-plus/tree/HEAD)
+Starting at version 1.2.0, the format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-[Full Changelog](https://github.com/CosmWasm/cw-storage-plus/compare/v1.0.1...HEAD)
+## [Unreleased]
 
-**Closed issues:**
+### Changed
 
-- implementation of KeyDeserialize for \(T, U\) is incorrect. [\#33](https://github.com/CosmWasm/cw-storage-plus/issues/33)
+- Update to `cosmwasm-std` 1.5.0 ([#60])
 
-**Merged pull requests:**
+[#60]: https://github.com/CosmWasm/cw-storage-plus/pull/60
+
+### Fixed
```

Comment:
I usually put all breaking changes unter `### Changed` where they are most likely expected

Reply:
In this case, both are breaking bug fixes, but I get what you mean. Visibility is probably more important here, so I'll put them in the `Changed` section.